fcrepo-kernel-modeshape/src/main/java/org/fcrepo/kernel/modeshape/identifiers/PathToNodeConverter.java

```diff
@@ -0,0 +1,65 @@
+package org.fcrepo.kernel.modeshape.identifiers;
+
+/*
+ * Licensed to DuraSpace under one or more contributor license agreements.
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership.
+ *
+ * DuraSpace licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except in
+ * compliance with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import javax.jcr.Node;
+import javax.jcr.RepositoryException;
+import javax.jcr.Session;
+
+import org.fcrepo.kernel.api.exception.RepositoryRuntimeException;
+
+/**
+ * 
+ * @author barmintor
+ *
+ */
+public class PathToNodeConverter extends IdentifierConverter<String, Node> {
```

Comment:
See comment above. I may be misunderstanding something here, but this doesn't seem like something that is converting identifiers to me.